lib/main.js

```diff
@@ -60,14 +59,6 @@ var defaults = {
   mmlNode: false,              // DOM element for MML output?
   svg: false,                     // return svg output?
   svgNode: false,              // DOM element for SVG output?
-
-  speakText: false,               // add spoken annotations to svg output?
```

Comment:
Your current implementation always returns `speakText` in the `results` object, and always adds some speech text to the SVG, HTML, or MML output.  I'd prefer to retain the option of adding speech text (though it could be true by default), and only add it and return it when requested.